build.sbt

```diff
@@ -75,6 +71,11 @@ libraryDependencies ++= Seq(
   "org.scalatestplus.play" %% "scalatestplus-play" % "7.0.1" % Test
 )
 
+dependencyOverrides ++= List(
+  // Play still uses an old version of jackson-core which has a vulnerability - https://security.snyk.io/vuln/SNYK-JAVA-COMFASTERXMLJACKSONCORE-7569538
+  "com.fasterxml.jackson.core" % "jackson-core" % "2.17.2"
```

Comment:
this is not a supported set up - jackson-databind and jackson-core should have the same version - I am a member of the Jackson dev team

Reply:
thanks for noting! We'll set jackson-databind to match

Reply:
One thing to watch out for is that Jackson 2.15+ has constraints and some of them have defaults that are high but not unlimited meaning that some edge case JSON may be lead to Jackson exceptions when reading or writing.

* https://www.javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.17.2/com/fasterxml/jackson/core/StreamReadConstraints.html
* https://www.javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.17.2/com/fasterxml/jackson/core/StreamWriteConstraints.html